docs/QUnit/module.md

```diff
@@ -199,10 +199,12 @@ QUnit.test( "makes a car", function( assert ) {
 
 ---
 
-Hooks stack on nested modules
+`before`/`beforeEach` hooks queue on nested modules. `after`/`afterEach` hooks stack on nested modules.
```

Comment:
This makes sense for those familiar with the data structures, but it might not be accessible to software engineers who don't have a strong background in data structures. Maybe this could be reworded to say outer and earlier-declared `before`/`beforeEach` functions run before​ inner and later-declared ones, and the reverse for `after`/`afterEach`? (Though I'll admit what I've proposed isn't great, either.)